### PR TITLE
Add section on catair mapping

### PR DESCRIPTION
**Pull request by OR13**
This PR adds a simple way to include additional html next to credentials with a specific tag.

### CODE REVIEW

**Comment by nissimsan** [comment]
@OR13, the attribute-level bottom-up approach hinted at here seems somewhat unrealistic. 

I suggest we rather start from the top. This document is a great starting point: 
https://www.cbp.gov/sites/default/files/assets/documents/2017-Oct/ACE_CATAIR_TableOfContents_June_2004.pdf

I would first map at credential-level to the CATAIR ToC. Then later on, we figure out how to break it further down appropriately.

File: docs/sections/catair-mapping.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    and how certain data elements might be new, 
+    and will have no mapping to CATAIR.
+  </p>
+
+  <table class="simple" >
+    <thead>
+      <tr>
+        <td>JSON Pointer</td>
+        <td>Value</td>
+        <td>CATAIR</td>
```
Comment (nissimsan) [inline docs/sections/catair-mapping.html:28]: Too detailed. Let's break this down before getting into attribute-level.

**Review by nissimsan — CHANGES_REQUESTED** [review]

File: docs/sections/catair-mapping.html (hunk 1 above)
Comment (OR13) [reply docs/sections/catair-mapping.html:28]: The original request was "per attribute", for "each credential".

**Review by mkhraisha — CHANGES_REQUESTED** [review]
My understanding is the requirement coming in is what are the fields that are NOT in CATAIR

File: docs/sections/catair-mapping.html (hunk 1 above)
Comment (nissimsan) [reply docs/sections/catair-mapping.html:28]: I see, I misunderstood. Then should we move this to the Credential section instead. That would make it clearer.

**Review by nissimsan — APPROVED** [review]

**Review by mkhraisha — CHANGES_REQUESTED** [review]
We should be able to inline HTML next to every credential instead of in its own section, for MTR this should be in the MTR credential area.

**Comment by OR13** [comment]
We discussed on the call, I will revise.

File: docs/openapi/components/schemas/credentials/MillTestReportCredential.yml [hunk 2]
```diff
@@ -4,6 +4,7 @@ $linkedData:
 title: Mill Test Report Credential
 tags:
   - Steel
+  - TechDemo2023
```
Comment (OR13) [inline docs/openapi/components/schemas/credentials/MillTestReportCredential.yml:7]: Add a tag like this.

File: docs/sections/catair/MillTestReportCredential.html [hunk 3]
```diff
@@ -0,0 +1,26 @@
+<table class="simple" >
```
Comment (OR13) [inline docs/sections/catair/MillTestReportCredential.html:1]: Add a file like this.

File: packages/traceability-schemas/scripts/schemas-to-vocab.js [hunk 4]
```diff
@@ -76,15 +80,25 @@ const buildClass = (schema) => {
 
   const depList = dependencies.length
     ? `<b>Dependencies</b><ul>${dependencies
-        .map((key) => `<li><a href='#${key}'>${key}</a></li>`)
+        .map((key) => `<li>${key}</li>`)
```
Comment (OR13) [inline packages/traceability-schemas/scripts/schemas-to-vocab.js:83]: this was causing all the respec errors... snippets had no valid anchor definition.

**Comment by OR13** [comment]
@mkhraisha please re-review.

Screen Shot 2023-05-16 at 6 17 27 PM

File: docs/openapi/components/schemas/credentials/MillTestReportCredential.yml (hunk 2 above)
Comment (mkhraisha) [reply docs/openapi/components/schemas/credentials/MillTestReportCredential.yml:7]: This should probably be a catair tag not a techdemo2023 tag

**Review by mkhraisha — COMMENTED** [review]
Probably a good idea to update the README with instructions on how to do this

**Comment by OR13** [comment]
@mkhraisha the entire script process to generate the spec is pretty gross... I would really like to refactor the whole thing, but that would be disruptive, here so I am holding off.

**Comment by nissimsan** [comment]
Neither @OR13  nor @mkhraisha  are on the call - leaving this PR for next week.

**Review by mkhraisha — APPROVED** [review]
Changes look great thank you!

**Pull request merged**